tests/browsercontext-page-event.spec.ts

```diff
@@ -184,3 +184,17 @@ it('should work with Ctrl-clicking', async ({ browser, server, isMac, browserNam
   expect(await popup.opener()).toBe(null);
   await context.close();
 });
+
+it('should not hang on ctrl-click during provisional load', async ({ context, page, server, isMac, browserName }) => {
+  it.info().annotations.push({ type: 'issue', description: 'https://github.com/microsoft/playwright/issues/11595' });
+  it.fixme(browserName === 'chromium', 'click hangs in Chromium waiting for selector');
```

Comment:
Make it.skip, because we'll never fix it. Chromium does not dispatch renderer messages while navigation is provisional.